package.json

```diff
@@ -48,6 +49,7 @@
     "husky": "^4.3.0",
     "lerna": "^3.22.1",
     "lint-staged": "^10.5.1",
+    "plop": "^2.7.6",
```

Comment:
Hmm. I'm wondering if this package should be here, at the root of the monorepo, or at `packages/ui/` since it's related to that. 

Any thoughts? 🤔

Reply:
Oh, [I see now](https://github.com/vtex/faststore/pull/1039/files#diff-d389cc1269e8b4c6b14b577978948d0380a8437dacb734fc319a4a66729e238bR36) it needs access to a previous directory file such as `themes/theme-b2c-tailwind`.